scripts/update.py

```diff
@@ -375,6 +376,66 @@ def format_df_for_d878(df: pd.DataFrame) -> pd.DataFrame:
 
     return df_878
 
+def format_df_for_icom(df: pd.DataFrame) -> pd.DataFrame:
+    """
+    Format a DataFrame of repeaters for use in Icom 878 CSV format.
+
+    Parameters
+    ----------
+    df : pd.DataFrame
+        All of the repeaters.
+
+    Returns
+    -------
+    pd.DataFrame
+        The Icom-compativle repeaters formatted for the Icom.
+        (Tested with Icom-705 - may work for others).
+    """
+
+    total_repeaters = len(df)
+
+    # Select FM and Fusion (in FM compat mode) channels.
+    df = df.loc[df["Mode"].isin(["FM", "NBFM", "Fusion"])].copy()
```

Comment:
The 705 can do DSTAR too, right ? Shall we add those later ?

Reply:
Yes! Are there dstar channels in the database?  I actually didn't notice
them.

Reply:
There are a handful. They don't take PL tones or anything specific, so assuming the existing code can deal with nulls or `""` under Tone, we should get them for free by including them on this line !

Reply:
I'd like to add this support in a followup PR, if that's OK.